src/schema/imported/index.js

```diff
@@ -5,6 +5,7 @@ import bookmarks from './bookmarks.json';
 import browserAction from './browser_action.json';
 import browserSettings from './browser_settings.json';
 import browsingData from './browsing_data.json';
+import captivePortal from './captive_portal.json';
```

Comment:
The tests are failing on not being able to find this by the looks of it.

Reply:
Indeed, I pushed the missing file in 2d7ccde3. Totally forgot :disappointed: